docs/index.html

```diff
@@ -23,59 +23,71 @@
 
 <div id="container" class="container">
   <header>
-    <h1>Open Wordnet Documentation</h1>
+    <h1 id="title">Open Wordnet Documentation</h1>
```

Comment:
This element is easily selectable without an id. Also see my comment below about the CSS for `#title`.